src/main/java/net/boomerangplatform/mongo/model/WorkflowQuotas.java

```diff
@@ -0,0 +1,44 @@
+package net.boomerangplatform.mongo.model;
```

Comment:
Move WorkflowQuotas from net.boomerangplatform.mongo.model. Only mongo models should be in this package